main/src/ca/uwaterloo/flix/language/phase/Reducer.scala

```diff
@@ -94,7 +94,8 @@ object Reducer {
         Expr.ApplyClo(e1, e2, ct, tpe, purity, loc)
 
       case Expr.ApplyDef(sym, exps, ct, tpe, purity, loc) =>
-        if (ct == ExpPosition.NonTail && Purity.isControlImpure(purity)) lctx.pcPoints += 1
```

Comment:
Kan du lave den her fix i sit eget PR? med at tjekke defn.expr.purity i stedet for purity. både her og i GenExpression